# Remove dependencies on user32.winxp.lib and user32-delay.dll in GYP
# user32.winxp.lib is deleted in source cleaning (which causes building to fail) and Google no longer supports Windows XP
# Also, this patch fixes a handle error that occurs when trying to run the build output

--- a/chrome/chrome_dll.gypi
+++ b/chrome/chrome_dll.gypi
@@ -142,7 +142,7 @@
               # Add a dependency to custom import library for user32 delay
               # imports only in x86 builds.
               'dependencies': [
-                'chrome_user32_delay_imports',
+                #'chrome_user32_delay_imports',
               ],
             },],
             ['OS=="win"', {
@@ -192,34 +192,34 @@
                     }],
                     ['target_arch=="ia32"', {
                       # Don't set an x64 base address (to avoid breaking HE-ASLR).
-                      'BaseAddress': '0x01c30000',
+                      #'BaseAddress': '0x01c30000',
                       # Link against the XP-constrained user32 import library
                       # instead of the platform-SDK provided one to avoid
                       # inadvertently taking dependencies on post-XP user32
                       # exports.
                       'AdditionalDependencies!': [
-                        'user32.lib',
+                        #'user32.lib',
                       ],
                       'IgnoreDefaultLibraryNames': [
-                        'user32.lib',
+                        #'user32.lib',
                       ],
                       # Remove user32 delay load for chrome.dll.
                       'DelayLoadDLLs!': [
-                        'user32.dll',
+                        #'user32.dll',
                       ],
                       'AdditionalDependencies': [
-                        'user32.winxp.lib',
+                        #'user32.winxp.lib',
                       ],
                       'DelayLoadDLLs': [
-                        'user32-delay.dll',
+                        #'user32-delay.dll',
                       ],
                       'AdditionalLibraryDirectories': [
-                        '<(DEPTH)/build/win/importlibs/x86',
+                        #'<(DEPTH)/build/win/importlibs/x86',
                       ],
                       'ForceSymbolReferences': [
                         # Force the inclusion of the delay load hook in this
                         # binary.
-                        '_ChromeDelayLoadHook@8',
+                        #'_ChromeDelayLoadHook@8',
                       ],
                     }],
                   ],
